methods.go

```diff
@@ -458,6 +458,38 @@ func (c *Client) GetAvailablePlugins() ([]string, error) {
 	return c.rpcWithStringsResult("core.get_available_plugins")
 }
 
+// EnablePlugin enables a plugin with the given name.
```

Comment:
Since it seems that this operation happens asynchronously on server side, shall we mention it in the GoDoc here?

(Same would be for `DisablePlugin`)